### PR TITLE
Fix issues when modifying the conditions of if and while statements

### PR DESCRIPTION
**Pull request by NatashaWso2**
Fixes https://github.com/ballerinalang/composer/issues/4596

### CODE REVIEW

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/if-node.jsx [hunk 1]
```diff
@@ -17,12 +17,15 @@
...
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 import CompoundStatementDecorator from './compound-statement-decorator';
 import TreeUtil from './../../../../../model/tree-util';
 import IfNodeModel from './../../../../../model/tree/if-node';
 import DropZone from './../../../../../drag-drop/DropZone';
 import DefaultNodeFactory from './../../../../../model/default-node-factory';
 import AddCompoundBlock from './add-compound-block';
+import FragmentUtils from './../../../../../utils/fragment-utils';
+import TreeBuilder from './../../../../../model/tree-builder';
```
Comment (pahans) [inline modules/web/js/ballerina/diagram2/views/default/components/nodes/if-node.jsx]: use absolute paths to import modules. https://github.com/ballerinalang/composer/wiki/Developer-best-practices#use-absolute-paths-to-import-modules

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/while-node.jsx [hunk 2]
```diff
@@ -18,33 +18,60 @@
...
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 import CompoundStatementDecorator from './compound-statement-decorator';
-import { getComponentForNodeArray } from './../../../../diagram-util';
 import WhileNodeModel from './../../../../../model/tree/while-node';
 import DropZone from './../../../../../drag-drop/DropZone';
 import './try-node.css';
+import FragmentUtils from './../../../../../utils/fragment-utils';
+import TreeBuilder from './../../../../../model/tree-builder';
```
Comment (pahans) [inline modules/web/js/ballerina/diagram2/views/default/components/nodes/while-node.jsx]: use absolute paths to import modules. https://github.com/ballerinalang/composer/wiki/Developer-best-practices#use-absolute-paths-to-import-modules

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/if-node.jsx (hunk 1 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/diagram2/views/default/components/nodes/if-node.jsx]: Fixed with f580f482cc82e92dd391ea225593c001ed2f5183

File: modules/web/js/ballerina/diagram2/views/default/components/nodes/while-node.jsx (hunk 2 above)
Comment (NatashaWso2) [reply modules/web/js/ballerina/diagram2/views/default/components/nodes/while-node.jsx]: Fixed with f580f482cc82e92dd391ea225593c001ed2f5183

**Pull request merged**